src/merkle/smt/full/mod.rs

```diff
@@ -172,6 +167,47 @@ impl Smt {
         <Self as SparseMerkleTree<SMT_DEPTH>>::insert(self, key, value)
     }
 
+    /// Compute what changes are necessary to insert the specified key-value pairs into this Merkle
+    /// tree, allowing for validation before applying those changes.
+    ///
+    /// This method returns a [`MutationSet`], which contains all the information for inserting
+    /// `kv_pairs` into this Merkle tree already calculated, including the new root hash, which can
+    /// be queried with [`MutationSet::root()`]. When you are done, call [`Smt::apply_mutations()`]
+    /// to commit these changes to the Merkle tree, dor [`drop()`] the [`MutationSet`] to discard
+    /// them.
```

Comment:
them -> it ?

Reply:
The subject is "these changes" — should I reword, or even just remove the bit about `drop()` entirely?

Reply:
I think it is fine to keep `them`.